casr/src/bin/casr-csharp.rs

```diff
@@ -70,6 +70,16 @@ fn main() -> Result<()> {
                 .value_name("PREFIX")
                 .help("Path prefix to strip from stacktrace and crash line"),
         )
+        .arg(
+            Arg::new("ld-preload")
+                .long("ld-preload")
+                .env("CASR_PRELOAD")
+                .action(ArgAction::Set)
+                .num_args(1..)
+                .value_name("LIBS")
+                .value_parser(clap::value_parser!(String))
+                .help("Set LD_PRELOAD for the target program without disrupting the CASR process itself")
```

Comment:
We need more clear help here about spaces and `:`

Reply:
Done